### ISSUE TITLE
using absolute path when adding include path

### ISSUE THREAD

**Issue by rhalstea** [p4lang/p4c#810]
I'm seeing an issue where I can't pass an absolute path the the -I argument, but passing a relative path works.

works:
``` $ p4test my_file.p4 -I include/ ```

fails:
``` $ p4test my_file.p4 -I $PWD/include/ ```

the error I get looks like
```
my_file.p4:2:31 fatal error: testdata_headers.p4: No such file or directory
 #include "testdata_headers.p4"
```

**Comment by hanw** [comment]
Hi @rhalstea, this may be related to one of the recent commit https://github.com/p4lang/p4c/commit/b4db551d5cb1106daf085baf547720f60905f7a4, I will look into it, thanks for submitting the issue.

**Comment by hanw** [comment]
@rhalstea I backed out commit b4db551 in #812, it should fix the issue you are seeing.

**Comment by hanw** [comment]
Fixed with 0531af6dd6c08626916f71a7a298a53c09c543c0.
